pkgs/applications/window-managers/i3/workstyle.nix

```diff
@@ -14,7 +14,11 @@ rustPlatform.buildRustPackage rec {
     sha256 = "0f4hwf236823qmqy31fczjb1hf3fvvac3x79jz2l7li55r6fd8hn";
   };
 
-  cargoSha256 = "1hy68wvsxncsy4yx4biigfvwyq18c7yp1g543c6nca15cdzs1c54";
+  # Latest rust release causes runtime crash, PR is pending:
+  # https://github.com/pierrechevalier83/workstyle/pull/12
```

Comment:
this is merged; can we switch to use a commit from upstream without a patch?

It looks like their `Cargo.toml` still specifies `version = "0.2.1"`; not sure if `-p0` or `unstable-...` is the proper convention here.

Reply:
nixpkgs prefers patches fetched with fetchpatch anyway.

Reply:
Yes, of course. Not sure why I did not fix that before opening this.